packages/http-client-python/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@typespec/http-client-python",
-  "version": "0.5.1",
+  "version": "0.5.2",
   "author": "Microsoft Corporation",
```

Comment:
Hi @iscai-msft , after version changes, I think there shall be update in package-lock.json. Please run install/build script then commit the file.